agent/stats/utils_windows.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package stats
+
+import (
+	"fmt"
+
+	"github.com/cihub/seelog"
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+// dockerStatsToContainerStats returns a new object of the ContainerStats object from docker stats.
+func dockerStatsToContainerStats(dockerStats *docker.Stats) (*ContainerStats, error) {
+	if numCores == uint64(0) {
+		seelog.Error("Invalid number of cpu cores acquired from the system")
+		return nil, fmt.Errorf("invalid number of cpu cores acquired from the system")
+	}
+
+	cpuUsage := dockerStats.CPUStats.CPUUsage.TotalUsage / numCores
```

Comment:
Can we verify that dividing by `numCores` is appropriate on Windows?  From when I was playing around with it on a 4-core machine, `docker stats` showed `100%` usage on Windows when using all 4 cores but showed `400%` on Linux for the same workload.

Reply:
Does it mean different things on each platform?

Reply:
@samuelkarp  confirmed in that case, the metrics will be the same as docker stats, it will show 100%.

Reply:
So do we need to divide by numCores or should we remove that?

Reply:
we should keep it, also this [code](https://github.com/moby/moby/blob/master/integration-cli/docker_api_stats_test.go#L50) indicates the same.

Reply:
As discussed offline: I don't have an objection to merging this as long as we close out on the right math before we release anything.